release-notes/v0.9.0.md

```diff
@@ -0,0 +1,200 @@
+# What's new in Visual Studio Code Java?
+
+*April 2020*
+
+Welcome to the April 2020 release of Java Extension Pack. This document aims to recap the features, bug fixes around Java language support in between releases. Here are some of the key highlights:
+
+- [Performance Improvements](#performance-improvements)
+- [Syntax Mode](#syntax-mode)
+- [Java 14](#java-14-supported)
+- [Call Hierarchy](#call-hierarchy)
+- [Data Breakpoint](#data-breakpoint-supported)
+- [Code Actions](#more-code-actions)
+- [SonarLint](#sonarlint)
+- [Work with JAR Files Directly](#work-with-jar-files-directly)
+- [Create New Class from File Explorer](#create-new-class-from-file-explorer)
+- [MicroProfile](#microprofile)
+- [Others](#others)
+
+## Performance Improvements
+
+The Java Language Server (the language server) has improved its speed of ***startup***, and the speed to ***import*** Maven projects. The team has been committed, and here are what they delivered so far.
+
+### Code Navigation in 10 Seconds
+
+Tired of waiting on the language server to start? Now the wait is over. With the help of the new Syntax Server, you will be able to use a fair amount of features within 10 seconds:
+
+![Syntax Server Startup](v0.9.0/syntax-server-startup.gif)
```

Comment:
gif will be ignore in the final vsix, see .vscodeignore. So remember to use the github absolute url to reference the screenshot.

Reply:
Sure!